internal/xds/extensions/extensions.gen.go

```diff
@@ -34,121 +34,28 @@ import (
-	_ "github.com/envoyproxy/go-control-plane/envoy/config/filter/network/direct_response/v2"
-	_ "github.com/envoyproxy/go-control-plane/envoy/config/filter/network/dubbo_proxy/v2alpha1"
-	_ "github.com/envoyproxy/go-control-plane/envoy/config/filter/network/echo/v2"
-	_ "github.com/envoyproxy/go-control-plane/envoy/config/filter/network/ext_authz/v2"
-	_ "github.com/envoyproxy/go-control-plane/envoy/config/filter/network/http_connection_manager/v2"
-	_ "github.com/envoyproxy/go-control-plane/envoy/config/filter/network/kafka_broker/v2alpha1"
-	_ "github.com/envoyproxy/go-control-plane/envoy/config/filter/network/local_rate_limit/v2alpha"
-	_ "github.com/envoyproxy/go-control-plane/envoy/config/filter/network/mongo_proxy/v2"
-	_ "github.com/envoyproxy/go-control-plane/envoy/config/filter/network/rate_limit/v2"
-	_ "github.com/envoyproxy/go-control-plane/envoy/config/filter/network/rbac/v2"
-	_ "github.com/envoyproxy/go-control-plane/envoy/config/filter/network/redis_proxy/v2"
-	_ "github.com/envoyproxy/go-control-plane/envoy/config/filter/network/sni_cluster/v2"
-	_ "github.com/envoyproxy/go-control-plane/envoy/config/filter/network/tcp_proxy/v2"
-	_ "github.com/envoyproxy/go-control-plane/envoy/config/filter/network/thrift_proxy/v2alpha1"
-	_ "github.com/envoyproxy/go-control-plane/envoy/config/filter/thrift/rate_limit/v2alpha1"
-	_ "github.com/envoyproxy/go-control-plane/envoy/config/filter/thrift/router/v2alpha1"
-	_ "github.com/envoyproxy/go-control-plane/envoy/config/filter/udp/udp_proxy/v2alpha"
-	_ "github.com/envoyproxy/go-control-plane/envoy/config/grpc_credential/v2alpha"
 	_ "github.com/envoyproxy/go-control-plane/envoy/config/grpc_credential/v3"
-	_ "github.com/envoyproxy/go-control-plane/envoy/config/health_checker/redis/v2"
-	_ "github.com/envoyproxy/go-control-plane/envoy/config/listener/v2"
 	_ "github.com/envoyproxy/go-control-plane/envoy/config/listener/v3"
-	_ "github.com/envoyproxy/go-control-plane/envoy/config/metrics/v2"
 	_ "github.com/envoyproxy/go-control-plane/envoy/config/metrics/v3"
-	_ "github.com/envoyproxy/go-control-plane/envoy/config/overload/v2alpha"
 	_ "github.com/envoyproxy/go-control-plane/envoy/config/overload/v3"
-	_ "github.com/envoyproxy/go-control-plane/envoy/config/ratelimit/v2"
 	_ "github.com/envoyproxy/go-control-plane/envoy/config/ratelimit/v3"
-	_ "github.com/envoyproxy/go-control-plane/envoy/config/rbac/v2"
 	_ "github.com/envoyproxy/go-control-plane/envoy/config/rbac/v3"
-	_ "github.com/envoyproxy/go-control-plane/envoy/config/resource_monitor/fixed_heap/v2alpha"
-	_ "github.com/envoyproxy/go-control-plane/envoy/config/resource_monitor/injected_resource/v2alpha"
-	_ "github.com/envoyproxy/go-control-plane/envoy/config/retry/omit_canary_hosts/v2"
-	_ "github.com/envoyproxy/go-control-plane/envoy/config/retry/omit_host_metadata/v2"
-	_ "github.com/envoyproxy/go-control-plane/envoy/config/retry/previous_hosts/v2"
```

Comment:
this lines seem not that right

Reply:
can you share why

Reply:
I'm thinking if there's a v3 api, it's safe to remove v2 version.

Reply:
ignore this, it's ok now, let's add them back if there's ture problem

Reply:
yeah I think its moved to another place
https://github.com/envoyproxy/go-control-plane/blob/main/envoy/extensions/retry/priority/previous_priorities/v3/previous_priorities_config.pb.go